test/component/visualrules_test.js

```diff
@@ -330,9 +330,10 @@ describe('VisualRules', function() {
             ], done);
         });
         it('should return an error when core-endpoint is not working', function(done) {
+            var rule = utilsT.loadExample('./test/data/good_vrs/visual_rule_1.json');
             utilsT.setServerCode(400);
             utilsT.setServerMessage('what a pity!');
-            clients.DeleteVR('a very strange rule to exist', function(error, data) {
+            clients.PutVR(rule.name, rule, function(error, data) {
```

Comment:
I understand that given that the fix is in a unit test it doesn't need CNR entry. Is that correct?

Just checking... :)